opencti-platform/opencti-graphql/tests/02-integration/01-database/rabbitmq-test.js

```diff
@@ -1,57 +1,52 @@
 import { expect, it, describe } from 'vitest';
 import { v4 as uuid } from 'uuid';
-import {
-  metrics,
-  pushToConnector,
-  registerConnectorQueues,
-  unregisterConnector,
-} from '../../../src/database/rabbitmq';
+import { getConnectorQueueDetails, metrics, purgeConnectorQueues, pushToConnector, registerConnectorQueues, unregisterConnector } from '../../../src/database/rabbitmq';
 import { CONNECTOR_INTERNAL_IMPORT_FILE } from '../../../src/schema/general';
 import { ADMIN_USER, testContext } from '../../utils/testQuery';
-import { RABBIT_QUEUE_PREFIX } from '../../../src/database/utils';
-
-describe('Rabbit basic and utils', () => {
```

Comment:
Note: this was a useless test since when metrics calls fails the test is passing anyway. It's now part of other test steps below in the same file.